src/components/App/App.js

```diff
@@ -1,27 +1,41 @@
 import './App.css';
-import movieData from "./movieData"
 import React from "react"
-import MoviesRepo from "./components/MoviesRepo/MoviesRepo"
-import SelectedMovie from "./components/SelectedMovie/SelectedMovie"
+import MoviesRepo from "../MoviesRepo/MoviesRepo"
+import SelectedMovie from "../SelectedMovie/SelectedMovie"
+import apiCalls from "../../apiCalls"
 
 class App extends React.Component {
     constructor() {
       super()
       this.state = {
-        movies: movieData.movies,
+        movies: [
+
+        ],
+      error: "",
         selectedMovie: ""
       }
     }
 
+    componentDidMount() {
+    apiCalls.fetchAPIData("/movies")
+    .then(response => {
+      if(typeof response === 'string') {
```

Comment:
This is very clever - I think I'm be doing all of my error handling like this in the future! It makes our App component much cleaner.